ls1intum/Artemis
# `Development`: Refine setup guide and update dockerfiles

### Checklist
#### General


- [x] I built the documentation locally.

### Description

Updating Atlassian Programming Exercise Setup Guide
- updating outdated information and dockerfiles
- making the guide more straightforward by adding more pictures and adding explanations for Windows users

### Steps for Testing


1. Build the documentation locally, using sphinx and executing `make html` in the `docs` folder and make sure there are no warnings
2. Read the documentation and question if the descriptions are understandable
3. Check the spelling and grammar

### Review Progress





#### Code Review
#### Manual Tests

### Screenshots
